data/himanshuprasad789 .json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Himanshu Prasad",
+  "type": "personal",
+  "bio": "Curiousity got me weaving the web, I am a fullstack developer",
+  "avatar":"https://media4.giphy.com/media/VbnUQpnihPSIgIXuZv/giphy.gif?cid=ecf05e47j7mu1kh87nu3b9078ym8vg84if4dunruylfrlyux&rid=giphy.gif&ct=g",
```

Comment:
Hi @himanshuprasad789, thank you for contributing  :tada: 
You can delete this line, it is not required anymore :+1: 
```Suggestion
```